alf/drivers/policy_driver.py

```diff
@@ -82,12 +82,16 @@ def __init__(self,
         # Somehow the restore_specs do not match restored metric tensors in
         # tensorflow_core/python/training/saving/functional_saver.py
         if training:
-            standard_metrics += [
-                tf_metrics.AverageReturnMetric(
-                    batch_size=env.batch_size, buffer_size=metric_buf_size),
-                tf_metrics.AverageEpisodeLengthMetric(
-                    batch_size=env.batch_size, buffer_size=metric_buf_size),
-            ]
+            batch_size = env.batch_size
+        else:
+            batch_size = 1
```

Comment:
I think this won't work. The checkpoint need the same batch size for play and train.

Reply:
Good point.  Tried and true.